File: core.shared/Net/DTO/V1/Auth/LoginRequest.cs
namespace Core.Shared.Net.DTO.V1.Auth
{
    public class LoginRequest
    {
        public string Login { get; set; }
        public string Password { get; set; }

        public string DeviceName { get; set; } = string.Empty;
        public string IpAddress { get; set; } = string.Empty;
        public string DeviceId { get; set; } = string.Empty;
        public bool isSiteWeb { get; set; } = false;
        public string Platform { get; set; } = string.Empty;
        public string Browser { get; set; } = string.Empty;
        public string Engine { get; set; } = string.Empty;
    }
}

File: shared-library/src/lib/models/authentication/request/Auth/LoginRequest.ts
export class LoginRequest {
    login: string;
    password: string;
    deviceName: string;
    ipAddress: string;
    deviceId: string;
    isSiteWeb: boolean;
    platform: string;
    browser: string;
    engine: string;

    constructor(
    ) {
        this.login = '';
        this.password = '';
        this.deviceName = '';
        this.ipAddress = '';
        this.deviceId = '';
        this.isSiteWeb = true;
        this.platform = '';
        this.browser = '';
        this.engine = '';
    }
}